clang/docs/RealtimeSanitizer.rst

```diff
@@ -84,6 +84,75 @@ non-zero exit code.
     #14 0x0001958960dc  (<unknown module>)
     #15 0x2f557ffffffffffc  (<unknown module>)
 
+Run-time flags
+--------------
+
+RealtimeSanitizer supports a number of run-time flags, which can be specified in the ``RTSAN_OPTIONS`` environment variable:
+
+.. code-block:: console
+
+   % RTSAN_OPTIONS=option_1=true:path_option_2="/some/file.txt" ./a.out
+   ...
+
+Or at compile-time by overloading the symbol ``__rtsan_default_options``:
```

Comment:
I don't think "overload" is technically the correct term. "provide"? Should we talk about visibility?

Reply:
Thanks for catching that, I agree "provide" is better.

Do you have any suggestions on how I might word a note on visibility?

Reply:
should we just add `__attribute__((__visibility__("default")))` for good measure? because i think if someone adds `-fvisibility=hidden` it won't work without this?

Reply:
I've also seen "override" as a term for a strong symbol being selected over a weak symbol - would that be appropriate here? We could say something like `Or at compile-time by overriding the weak symbol ``__rtsan_default_options`` `. I also like making the visibility attribute explicit on the code block 👍

Reply:
Do users care that this is overriding a weak symbol?

Reply:
Perhaps not, I just thought it might add some helpful context. I don't feel strongly about including it